src/components/helpers/list_videos.ts

```diff
@@ -7,8 +7,17 @@ export function listVideos(item: number) {
 		'https://pipedapi.scoopy.zone/feed/unauthenticated/rss?channels=UCeeFfhMcJa1kjtfZAGskOCA',
 		'https://pipedapi.scoopy.zone/feed/unauthenticated/rss?channels=UC3KEoMzNz8eYnwBC34RaKCQ',
 		'https://pipedapi.scoopy.zone/feed/unauthenticated/rss?channels=UCYs3kVmLLfbaRL6Af2ELMHg',
+		'https://pipedapi.scoopy.zone/rss/playlists/PLAbMhAYRuCUhawCEV2oXZGrienoKTN16X',
```

Comment:
This is for feeds like ZeroPunc and Pitch Meetings, yeah?

Reply:
pitch actually split out into its own channel, but prior to that happening yeah, and yeah, this one is specifically zeropunc